### PR TITLE
Removed proxy from conf per #1227

### LINKED ISSUE #1227: Remove client proxy config from reference.conf
Until we actually implement #115 / #583 it's confusing we have some configuration for client-side HTTP proxy'ing while that feature is not actually implemented.

We should probably remove it from reference.conf until the feature is in fact there.

### PR DESCRIPTION
**Pull request by Scalahansolo**
Fixes #1227

### CODE REVIEW

**Comment by jlprat** [comment]
PLS BUILD

**Comment by jlprat** [comment]
OK TO TEST

File: akka-http-core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -291,38 +291,6 @@ akka.http {
...
-        #   port = 8080
-        #   non-proxy-hosts = ["*.direct-access.net"]
-        # }
-        http = default
-
-        # Proxy settings for HTTPS requests (currently unsupported)
-        https = default
-      }
-
-      # Socket options to set for the listening socket. If a setting is left
```
Comment (jrudolph) [inline akka-http-core/src/main/resources/reference.conf]: I think these are still needed.

**Comment by raboof** [comment]
PLS BUILD

Test failure might be the instability (instabilities?) tracked in #299

**Comment by jrudolph** [comment]
Thanks, @Scalahansolo!

**Pull request merged**